src/applications/personalization/profile-2/components/connected-apps/components/ConnectedApp.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import moment from 'moment';
+
+import PropTypes from 'prop-types';
+import AdditionalInfo from '@department-of-veterans-affairs/formation-react/AdditionalInfo';
+import recordEvent from 'platform/monitoring/record-event';
+
+import { AccountModal } from './AccountModal';
+
+class ConnectedApp extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { modalOpen: false, detailsOpen: false };
+  }
+
+  closeModal = () => {
+    this.setState({ modalOpen: false });
+  };
+
+  openModal = () => {
+    this.setState({ modalOpen: true });
+  };
+
+  confirmDelete = () => {
+    recordEvent({
+      event: 'account-navigation',
+      'account-action': 'disconnect-button',
+      'account-section': 'connected-accounts',
+    });
+    this.props.confirmDelete(this.props.id);
+    this.closeModal();
+  };
+
+  toggleDetails = () => {
+    this.setState({ detailsOpen: !this.state.detailsOpen });
+  };
+
+  render() {
+    const { logo, title, created, grants } = this.props.attributes;
+
+    return (
+      <div
+        className="border-box vads-u-display--flex vads-u-align-items--flex-start vads-u-padding--3 vads-u-border-color--gray-lightest vads-u-border--2px
+        vads-u-margin-y--2"
+      >
+        <img
+          className="va-connected-acct-account-logo vads-u-margin-right--2p5"
+          src={logo}
+          alt={`${title} logo`}
+        />
+        <div className="vads-u-flex--4 vads-u-align-items--flex-start">
+          {/* ON MOBILE CLASS BELOW NEEDS TO BE FLEX DIRECTION COLUMN */}
+          <div className="vads-u-display--flex vads-u-justify-content--space-between vads-u-flex-direction--column medium-screen:vads-u-flex-direction--row">
+            <div>
+              <h2 className="vads-u-margin--0 vads-u-color--gray-dark">
+                {title}
+              </h2>
+              <p className="vads-u-margin-top--0p5">
+                Connected on {moment(created).format('MMMM D, YYYY h:mm A')}
+              </p>
+            </div>
+
+            <button
+              aria-label={`Disconnect ${title} from your account`}
+              className="usa-button-secondary"
+              onClick={this.openModal}
+            >
+              Disconnect
+            </button>
+          </div>
+
+          <AccountModal
+            deleting={this.props.deleting}
+            appName={title}
+            modalOpen={this.state.modalOpen}
+            onCloseModal={this.closeModal}
+            onConfirmDelete={this.confirmDelete}
+          />
+          <AdditionalInfo triggerText={`Learn about ${title}`}>
+            <p>
+              <strong>{title}</strong>
+              &nbsp;can view your:
+            </p>
+            <ul>
+              {grants && grants.map((a, idx) => <li key={idx}>{a.title}</li>)}
```

Comment:
nit: maybe wrap the entire `ul` in the `{grants` check so:
- you don't need do the check here
- the `ul` isn't shown at all if the `grants` is empty
But I'm guessing that `grants` array is basically guaranteed to have stuff in it